channel/src/main/java/com/synopsys/integration/alert/channel/azureboards2/AzureBoardsAlertIssuePropertiesManager.java

```diff
@@ -0,0 +1,71 @@
+ */
+package com.synopsys.integration.alert.channel.azureboards2;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import org.jetbrains.annotations.Nullable;
+
+import com.synopsys.integration.alert.channel.api.issue.model.IssueBomComponentDetails;
+import com.synopsys.integration.alert.channel.api.issue.model.IssuePolicyDetails;
+import com.synopsys.integration.alert.channel.api.issue.model.ProjectIssueModel;
+import com.synopsys.integration.alert.channel.azure.boards.service.AzureCustomFieldManager;
+import com.synopsys.integration.alert.common.message.model.LinkableItem;
+import com.synopsys.integration.azure.boards.common.service.workitem.request.WorkItemElementOperation;
+import com.synopsys.integration.azure.boards.common.service.workitem.request.WorkItemElementOperationModel;
+import com.synopsys.integration.azure.boards.common.util.AzureFieldDefinition;
+
+public class AzureBoardsAlertIssuePropertiesManager {
+    private static final String POLICY_ADDITIONAL_KEY_COMPATIBILITY_LABEL = "Policy Violated";
+    private static final String CATEGORY_TYPE_POLICY_COMPATIBILITY_LABEL = "Policy";
+    private static final String CATEGORY_TYPE_VULNERABILITY_COMPATIBILITY_LABEL = "Vulnerability";
+
+    public List<WorkItemElementOperationModel> createWorkItemRequestCustomFieldOperations(ProjectIssueModel alertIssueSource) {
+        LinkableItem provider = alertIssueSource.getProvider();
+        String providerKey = AzureBoardsSearchPropertiesUtils.createProviderKey(provider.getLabel(), provider.getUrl().orElse(null));
+        String topicKey = AzureBoardsSearchPropertiesUtils.createNullableLinkableItemKey(alertIssueSource.getProject());
+        String subTopicKey = AzureBoardsSearchPropertiesUtils.createNullableLinkableItemKey(alertIssueSource.getProjectVersion().orElse(null));
+
+        IssueBomComponentDetails bomComponentDetails = alertIssueSource.getBomComponentDetails();
+        String componentKey = AzureBoardsSearchPropertiesUtils.createNullableLinkableItemKey(bomComponentDetails.getComponent());
+        String subComponentKey = AzureBoardsSearchPropertiesUtils.createNullableLinkableItemKey(bomComponentDetails.getComponentVersion().orElse(null));
+
+        List<WorkItemElementOperationModel> customFields = new ArrayList<>(7);
+        addStringField(customFields, AzureCustomFieldManager.ALERT_PROVIDER_KEY_FIELD_REFERENCE_NAME, providerKey);
+        addStringField(customFields, AzureCustomFieldManager.ALERT_TOPIC_KEY_FIELD_REFERENCE_NAME, topicKey);
+        addStringField(customFields, AzureCustomFieldManager.ALERT_SUB_TOPIC_KEY_FIELD_REFERENCE_NAME, subTopicKey);
+        addStringField(customFields, AzureCustomFieldManager.ALERT_COMPONENT_KEY_FIELD_REFERENCE_NAME, componentKey);
+        addStringField(customFields, AzureCustomFieldManager.ALERT_SUB_COMPONENT_KEY_FIELD_REFERENCE_NAME, subComponentKey);
+
+        String categoryKey;
```

Comment:
I think setting this equal to CATEGORY_TYPE_VULNERABILITY_COMPATIBILITY_LABEL at the beginning is easier to read than using an if else later on.